### ISSUE TITLE
Module and elim

### ISSUE THREAD

**Issue by thomas-lamiaux** [agda/cubical#822]
@felixwellen realized last week that there is files such as the [Free comMonoid](https://github.com/agda/cubical/blob/master/Cubical/HITs/FreeComMonoids/Base.agda) or [Direct Sum](https://github.com/agda/cubical/blob/master/Cubical/Algebra/DirectSum/Base.agda)
that defines a module name then a function f. 
So one need to call `Elim.f X Y Z x y z` which is more complicated and not very clear.
I think we decided that it should be done by replacing the module name by _ and f by the module name ?

**Comment by felixwellen** [comment]
I agree that it would be good to change all instances of that in a PR.
